perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -54,6 +80,107 @@
                   'num_machines': 1}
 
 
+JOB_FILE_TEMPLATE = """
```

Comment:
Sgtm. Yes these are comparable job files so the tests should be comparable
too.

Thanks for the explanation.

On Aug 20, 2015, at 9:42 PM, noahl <notifications@github.com> wrote:

In perfkitbenchmarker/benchmarks/fio_benchmark.py
<https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/421#discussion_r37606266>
:

> @@ -54,6 +80,107 @@
>                    'num_machines': 1}
>
>
> +JOB_FILE_TEMPLATE = """

@stfeng <https://github.com/stfeng>

I agreed that it was a new benchmark because I thought the results from the
new code wouldn't be comparable to results from the old code. I just
checked the job files, and actually the old and new use the same options (I
think I did this on purpose), so they are more comparable than I thought.

The new jobfile generation code will test directly against the device if
you use the --against_device argument.

—
Reply to this email directly or view it on GitHub
<https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/421/files#r37606266>
.